borg/archiver.py

```diff
@@ -56,6 +56,9 @@ def print_verbose(self, msg, *args, **kw):
         msg = args and msg % args or msg
         logger.info(msg)
 
+    def print_status(self, status, path):
+        logger.debug("%1s %s", status, remove_surrogates(path))
```

Comment:
well, i was hoping to expand this with better filtering rules as per #302, so i moved it to a different function so functionality is grouped logically. furthermore, status output was handled differently in the two locations, which seems like a bad idea: refactoring fixed that issue.

as for the log level, yes, it's a policy change: i found that just one `-v` shouldn't flood the terminal... i can revert that if you prefer, but it does make `-v` actually usable here. :)